balances.json

```diff
@@ -3,5 +3,6 @@
   "project": "idseq",
   "env": "staging",
   "name": "web",
-  "region": "us-west-2"
+  "region": "us-west-2",
+  "resque_queues": "data_migration,precache_report_info,q03_pipeline_run,transfer_basespace_files,generate_bulk_download,*"
```

Comment:
what is `balances.json` for and why not put the value in `czecs-resque.json` directly?